dev/source/docs/ros-distance-sensors.rst

```diff
@@ -0,0 +1,72 @@
+.. _ros-distance-sensors:
+
+==========================
+ROS distance sensors usage
+==========================
+
+.. warning::
+
+    Distance sensors message are currently support on :
+
+    - Ardupilot Master
+    - Copter >= 3.5.1
```

Comment:
You should then put the versions separately in Rangefinder and Proximity sections. And if it isn't supported yet in some vehicle it shouldn't be mentioned, otherwise it's possible those versions will get out without support for it and we'll get people complaining about it.